lib/portlayer/storage/volume.go

```diff
@@ -70,10 +71,12 @@ type Volume struct {
 
 	// Backing device
 	Device Disk
+
+	CopyMode executor.CopyMode
 }
 
 // NewVolume creates a Volume
-func NewVolume(store *url.URL, ID string, info map[string][]byte, device Disk) (*Volume, error) {
+func NewVolume(store *url.URL, ID string, info map[string][]byte, device Disk, copyMode executor.CopyMode) (*Volume, error) {
```

Comment:
`CopyNever` is the default, `CopyNew` has a value 1 

Changed the enum to be iota+1 by default as requested by @caglar10ur below